lib/util.js

```diff
@@ -1,8 +1,14 @@
 /*
  * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
  */
-import {asn1, oids, util} from 'node-forge';
-const {ByteBuffer} = util;
+
+const DER_PRIVATE_KEY_PREFIX = Buffer.from([
+  0x30, 0x2e, 0x02, 0x01, 0x00, 0x30, 0x05, 0x06,
+  0x03, 0x2b, 0x65, 0x70, 0x04, 0x22, 0x04, 0x20
+], 'hex');
```

Comment:
This isn't hex, so it isn't needed. `hex` would be this:

```
Buffer.from('302e020100300506032b657004220420', 'hex');
```

Reply:
still doesn't fit on a single line, but here you go: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/5/commits/84c4168f90c5e8494267320c091f6f5af8246520